README.md

```diff
@@ -202,6 +205,8 @@ maintained by [Parity Technologies](https://www.parity.io/). Source code availab
 - [TxWrapper](https://github.com/paritytech/txwrapper) - Helpful library for offline transaction creation.
 - [VSCode Substrate](https://marketplace.visualstudio.com/items?itemName=paritytech.vscode-substrate) - Plugin for Visual Studio Code.
 - [Polkaholic.io](https://polkaholic.io) - Multi-chain block explorer with API and DeFi support across 40+ parachains.
+- [SubID](https://sub.id) - The #1 portfolio management tool in the Polkadot ecosystem. See your balances, crowdloan history, NFTs, and more. Claim vested tokens and perform cross-chain transfers.
```

Comment:
```suggestion
- [SubID](https://github.com/dappforce/subid) - An advanced cross-chain portfolio management tool for the Polkadot ecosystem, allowing any user to see their balances across chains; view their crowdloan history; view their NFTs across Polkadot ecosystem chains; claim their vested tokens and perform cross-chain transfers.
```

Although a big fan of this tool myself, I would avoid putting "#1" for sake of un-bias compared to other projects here who could also claim to be #1 in their field. And linking to the Github repo instead for anyone interested in forking or building with it as a tool. The `sub.id` URL can be found there too. Great addition to this repo though 👍🏻

Reply:
Sure ✌🏼